src/config.ts

```diff
@@ -128,6 +128,7 @@ const DEFAULTS = o({
         "<S-Insert>": "mode ignore",
         "<CA-Esc>": "mode ignore",
         "<CA-`>": "mode ignore",
+        "<Esc>": "composite mode normal ; hidecmdline",
```

Comment:
What purpose does `mode normal` serve here? It seems tautological as the bind can only ever be used in normal mode.